### ISSUE TITLE
Teams at the Minimap

### ISSUE THREAD

**Issue by TheRaffy** [OpenRA/OpenRA#10499]
I suggest do additionally draw the team number of the players on the minimap.
In my opinion this would speed up the match making and avoid games with twisted teams.

The picture is just a mock-up. and shall help to show what I mean. 
Feel free to feedback.

![minimap idea](https://cloud.githubusercontent.com/assets/5147014/12274572/bcd4558a-b96b-11e5-8064-7aebc43d6d65.png)

**Comment by pchote** [comment]
IMO #5740 is a cleaner (less visually cluttered) way of displaying the team balance.

**Comment by TheRaffy** [comment]
Compared to the other idea, you don't see who's in the wrong team immediately. That's better here.

**Comment by ShadowA** [comment]
(Another possible idea : 
Add a button "Show teams". 
It changes the letters to the number of the team. 
And each team has one color (ex. blue vs red [vs green vs ...])

**Comment by TheRaffy** [comment]
ya in the end who cares what name the spot has.

**Comment by penev92** [comment]
What I don't understand is in what world is something like this needed?

**Comment by TheRaffy** [comment]
you, don't know the pain of waiting in the lobby and after 10 minutes its full house and then the complaining starts..about wrong teams get ready and so on.

**Comment by epicelite** [comment]
Yes this is much needed, but this isn't a visually pleasing solution.
